flink-table/flink-sql-client/pom.xml

```diff
@@ -511,6 +511,12 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-sql-parser</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Take a look at the shade plugin below. We can do as table-planner module append the parser class into the final jar. But I think we don't need all classes in the parser jar and we can filter out if it is not needed.

Reply:
Added configuration to the shade plugin.  I pushed this commit and see if it works.

Reply:
> Added new configuration to shade plugin. I pushed a new commit to see if it works.

It works.